modules/lib/lib-auditlog/src/main/resources/lib/xp/auditlog.ts

```diff
@@ -23,26 +23,26 @@ function checkRequired<T extends object>(obj: T, name: keyof T): void {
     }
 }
 
-export interface AuditLogParams {
+export interface AuditLogParams<DATA extends Record<string, unknown> = Record<string, unknown>> {
```

Comment:
The convention used other places in the code is that type parameters are capitalized (not upper case).

```suggestion
export interface AuditLogParams<Data extends Record<string, unknown> = Record<string, unknown>> {
```

Reply:
Where is it smart to have default type parameters (` = Record<string, unknown>`)? I'm thinking maybe on the functions, and not in the interfaces. Specially when the objects are only created by this library.

(This goes for the other cases below too.)

What do you think?